clj/resources/lang/en.edn

```diff
@@ -510,6 +510,9 @@
    :slipstream.ui.models.parameters.launch-mutable-run?.description             :global.nouns.mutable-deployment
    :slipstream.ui.models.parameters.launch-mutable-run?.help-hint               "Enabling this option will launch a _mutable_ deployment, which will allow the `multiplicity` for each node to be changed after launch. **Warning!** This is a _BETA_ feature!"
 
+   :slipstream.ui.models.parameters.image-target-cloud.description              :global.nouns.cloud
+   :slipstream.ui.models.parameters.image-target-cloud.help-hint                "Choose the cloud to deploy the image to. The cloud flagged with a `*` is the default configured in your user profile."
+
    :slipstream.ui.models.parameters.deployment-target-cloud.description         :global.nouns.cloud
    :slipstream.ui.models.parameters.deployment-target-cloud.help-hint           "Choose the cloud to deploy to. The cloud flagged with a `*` is the default configured in your user profile. The option `Specify for each node` will reveal this menu for each of the nodes below."
```

Comment:
@loomis 
Thanks for the correction above.
What about the equivalent string for the deployments: `Choose the cloud to deploy to.` ?
=> `Choose the cloud where the deployment will be deployed.` ? :unamused: